specification/security/resource-manager/Microsoft.Security/stable/2019-01-01/security.json

```diff
@@ -734,6 +983,283 @@
+                                "description": "No supported regulatory compliance data for the standard"
+                            }
+                        ]
+                    }
+                },
+                "passedControls": {
+                    "readOnly": true,
+                    "type": "integer",
+                    "description": "The number of supported regulatory compliance controls of the given standard with a passed state"
+                },
+				"failedControls": {
+                    "readOnly": true,
+                    "type": "integer",
+                    "description": "The number of supported regulatory compliance controls of the given standard with a failed state"
+                },
+				"skippedControls": {
+                    "readOnly": true,
+                    "type": "integer",
+                    "description": "The number of supported regulatory compliance controls of the given standard with a skipped state"
+                },
+				"unsupportedControls": {
+                    "readOnly": true,
+                    "type": "integer",
+                    "description": "The number of regulatory compliance controls of the given standard which are unsupported by automated assessments"
+                }
+            }
+        },
+		"RegulatoryComplianceControlList": {
+			"description": "List of regulatory compliance controls response",
+			"required": [
+				"value"
+            ],
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "description": "List of regulatory compliance controls",
+                    "items": {
+                        "$ref": "#/definitions/RegulatoryComplianceControl"
+                    }
+                },
+                "nextLink": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The URI to fetch the next page."
+                }
+            }
+        },
+        "RegulatoryComplianceControl": {
+            "type": "object",
+            "description": "Regulatory compliance control details and state",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "description": "Regulatory compliance control data",
+                    "$ref": "#/definitions/RegulatoryComplianceControlProperties"
+                }
+            },
+            "allOf": [{
+                "$ref": "#/definitions/Resource"
+            }]
+        },
+        "RegulatoryComplianceControlProperties": {
+		 "type": "object",
+            "description": "Regulatory compliance control data",
+            "properties": {
+                "description": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The description of the regulatory compliance control"
+                },
+				"standardName": {
```

Comment:
Not needed to be added as a property. It is present in the URL and the id in the response.